File: 프로그래머스/1/42748. K번째수/K번째수.py
# def solution(array, commands):
#     #0934-0943
    
#     answer = []
#     for j in commands:
#         splitArr=array[j[0]-1:j[1]]
#         splitArr.sort()
#         answer.append(splitArr[j[2]-1])
#     return answer

def solution(array, commands):
    answer=[]
    for i in commands:
        tmp= array[i[0]-1:i[1]].copy()
        tmp.sort()
        answer.append(tmp[i[2]-1])
    return answer

File: 프로그래머스/1/42748. K번째수/K번째수.js
function solution(array, commands) {
    var answer = [];
    
    for( i of commands){
        let arr2=[...array]
        let tmp=arr2.slice(i[0]-1,i[1]).sort((a,b)=>a-b)
        // console.log(tmp)
        answer.push(tmp[i[2]-1])
        
    }
    return answer;
}